projects/woff2/Readme.md

```diff
@@ -0,0 +1,51 @@
+# woff2
+
+woff2 is a C library for converting woff2 files into ttf and back.
+
+## Build Docker
+
+    $ sudo docker build -t oss-sydr-fuzz-woff2 .
+
+## Run Hybrid Fuzzing
+
+Unzip Sydr (`sydr.zip`) in `projects/woff2` directory:
+
+    $ unzip sydr.zip
+
+## Run docker:
+
+    $ sudo docker run --cap-add=SYS_PTRACE  --security-opt seccomp=unconfined -v /etc/localtime:/etc/localtime:ro --rm -it -v $PWD:/fuzz oss-sydr-fuzz-woff2 /bin/bash
+
+## Change directory to `/fuzz`:
+
+    # cd /fuzz
+
+## Run hybrid fuzzing with libfuzzer:
+
+    # sydr-fuzz -c convert_woff2ttf_fuzzing.toml run
+
+## Get LCOV HTML coverage report:
```

Comment:
Update this section and add sections about security predicates and CASR (like here: https://github.com/ispras/oss-sydr-fuzz/commit/2be6b90edeedd00ea00e7e7c60dcc92e1f5bde39#diff-0e4a54e332ec6dfe54b0cc05c4192ac7b54255a880e65b6c6ea2bd2c46989552R31). Libjpeg contains older version of README